### PR TITLE
fix(headerflyout): remove caret icon

### PR DESCRIPTION
**Pull request by alexkrolick**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/features/header-flyout/HeaderFlyout.js [hunk 1]
```diff
@@ -30,10 +30,15 @@ class HeaderFlyout extends React.Component<Props> {
...
 
     render() {
-        const { header, footer, flyoutButton, children, scrollRefFn, ...rest } = this.props;
+        const { header, footer, flyoutButton, children, scrollRefFn, className, ...rest } = this.props;
 
         return (
-            <Flyout closeOnClick={false} offset={HeaderFlyout.panelOffset} {...rest}>
+            <Flyout
+                closeOnClick={false}
+                offset={HeaderFlyout.panelOffset}
```
Comment (booc0mtaco) [inline src/features/header-flyout/HeaderFlyout.js]: When opened, the offset should now align the flyout vertically with the button clicked. with the caret, it was offset by a little bit. If I remember correctly, we need to shift this over as well

Comment (alexkrolick) [reply src/features/header-flyout/HeaderFlyout.js]: Screen Shot 2019-03-15 at 12 20 39 PM

**Comment by alexkrolick** [comment]
@booc0mtaco amended commit name

**Pull request merged**